plugins/inputs/radius/radius.go

```diff
@@ -0,0 +1,104 @@
+package radius
+
+import (
+	"context"
+	_ "embed"
+	"net"
+	"sync"
+	"time"
+
+	"layeh.com/radius"
+	"layeh.com/radius/rfc2865"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Radius struct {
+	Servers         []string
```

Comment:
Please add the TOML tags here
```suggestion
type Radius struct {
	Servers         []string  `toml:"servers"`
```
same for the other (exported) fields. For `Log` you should do
```golang
Log             telegraf.Logger `toml:"-"`
```
 to prevent the variable to be set from the config by accident.